python/paddle/fluid/tests/unittests/optim.py

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
```

Comment:
Put all these contents to `dist_transformer.py` to avoid too many files under `unitests` folder.

Reply:
Done.